Source/Scene/Scene.js

```diff
@@ -945,6 +950,40 @@ define([
         },
 
         /**
+         * This property is for debugging only; it is not for production use.
+         * <p>
+         * When <code>true</code>, draws primitives to show the boundaries of the camera frustum
```

Comment:
Tweak wording a bit - `draws outlines to show the boundaries of the camera frustums`